unixfs/io/dirbuilder.go

```diff
@@ -29,22 +27,13 @@ func NewDirectory(dserv mdag.DAGService) *directoryBuilder {
 }
 
 // AddChild adds a (name, key)-pair to the root node.
-func (d *directoryBuilder) AddChild(name string, k key.Key) error {
-	// TODO(cryptix): consolidate context managment
-	ctx, cancel := context.WithTimeout(context.TODO(), time.Minute)
```

Comment:
no sure if `WithTimeout` should be removed.
In other places sometimes it is 30 seconds or a minute, or no `WithTimeout` at all.